.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,18 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Build the Docker image
+      run: docker build . --file Dockerfile --tag my-image-name:$(date +%s)
```

Comment:
```suggestion
  build:

    runs-on: ubuntu-latest

    steps:
      - uses: actions/checkout@v4

      - name: Set up Docker Buildx
        uses: docker/setup-buildx-action@v2

      - name: Login to GitHub Container Registry
        if: ${{ github.ref == 'refs/heads/master' }}
        uses: docker/login-action@v2
        with:
          registry: ghcr.io
          username: ${{ github.repository_owner }}
          password: ${{ secrets.GITHUB_TOKEN }}

      - name: Build Docker image (and push on main)
        uses: docker/build-push-action@v4.0.0
        with:
          push: ${{ github.ref == 'refs/heads/master' }}
          tags: ghcr.io/cthit/digit.chalmers.it:latest
```
Thoughts on this?

Reply:
Looks great, I honestly just used the default Action for building, but this will upload the image to ghcr as well right?